ports/stm32/mboot/README.md

```diff
@@ -137,6 +137,42 @@ The firmware to load must be a gzip'd DfuSe file (.dfu.gz).
 The provided fwupdate.py script contains helper functions to call into Mboot
 with the correct data, and also to update Mboot itself.
 
+Signed and Encrypted DFU support
+--------------------------------
+
+Mboot optionally supports signing and encrypting the binary in the dfu file. 
+This requires additional settings in the board config and requires `pyhy` python 
+module to be installed into `python3` used when building mboot, eg:
+
+    pip3 install pyhy
+
+In addition to the changes made to mpconfigboard.mk earlier, for encrypted 
+support you also need to add:
+
+    MBOOT_ENCRYPTED = 1
+
+Also, alongside the existing `TEXT0_ADDR` setting the length of application 
+flash space is needed, so for an F767 with 2MB of flash where the application 
+starts after the 32K of mboot space, it will equal `2048K - 32K == 0x1F8000`:
+
+    TEXT0_ADDR = 0x08008000
+    TEXT0_LENGTH = 0x1F8000
+
```

Comment:
Should add a comma:

```suggestion
Also, alongside the existing `TEXT0_ADDR` setting, the length of application 
```

Also, it would be helpful to include _why_ `TEXT0_LENGTH` is necessary. Something along the lines of: "`TEXT0_LENGTH' is required to indicate where to store the checksum required to validate the encrypted binary".